common/brave_channel_info.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/brave_channel_info.h"
+
+#include "chrome/common/channel_info.h"
+
+namespace brave {
+
+std::string GetChannelName() {
+#if defined(OFFICIAL_BUILD)
+  std::string channel_name = chrome::GetChannelName();
+  if (channel_name.empty()) {
+    channel_name = "release";
+  }
+
+  return channel_name;
+#else  // OFFICIAL_BUILD
+  return "development";
```

Comment:
cc @aekeus @bridiver it seems like before we were returning release for both non official and official builds. It might be good to add a selector for development for stats.

Reply:
The 'developer' channel is available on stats in the database. We would need to add the 'development' identifier to a few drop down menus. This would not be a large change to make.

Reply:
I would suggest 'developer' instead of 'development' as that exists on stats already.

```select * from dtl.channels ;
  channel  |     description     |      label
-----------+---------------------+------------------
...
developer | Developer channel   | Developer
...```

Reply:
Changed `development` `developer`, thanks